src/index.html

```diff
@@ -1,13 +1,441 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Dia</title>
+    <title>Air</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@400;600&family=Poppins:wght@600;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="icon" href="/images/air-logo.svg">
   </head>
   <body>
-    <h1>Dia</h1>
-    <script type="text/javascript" src="scripts/main.js"></script>
+    <header class="header">
+      <article class="header__wrapper">
+        <div class="header__top">
+          <div class="header__content">
+            <a href="#" class="header__logo-link">
+              <img
+              src="./images/air-logo.svg"
+              alt="Air logo"
+              class="header__logo"
+              >
+            </a>
+
+            <a href="#menu" class="burger-menu"></a>
+
+            <nav class="nav">
+              <ul class="nav__list">
+                <li class="nav__item">
+                  <a class="nav__link" href="#about-us">About us</a>
+                </li>
+
+                <li class="nav__item">
+                  <a class="nav__link" href="#services">Services</a>
+                </li>
+
+                <li class="nav__item">
+                  <a class="nav__link" href="#testimonials">Testimonials</a>
+                </li>
+
+                <li class="nav__item">
+                  <a class="nav__link" href="#contact-us">Contact us</a>
+                </li>
+              </ul>
+            </nav>
+          </div>
+
+          <a href="#send-us-a-message" class="header__button-link">
+            <button class="header__button">Hire us</button>
+          </a>
+
+          <aside class="nav-mobile page__menu" id="menu">
+            <div class="nav-mobile__container">
+              <div class="nav-mobile__top">
+                <a href="#" class="nav-mobile__logo-link">
+                  <img
+                    src="images/air-logo.svg"
+                    alt="Air logo"
+                    class="nav-mobile__logo"
+                  >
+                </a>
+                <a href="#" class="nav-mobile__icon--close"></a>
+              </div>
+
+              <div class="nav-mobile__bottom">
```

Comment:
This div is most likely redundant, try removing it and just adding this or similar class to the below nav element